lib/galaxy/jobs/runners/util/job_script/DEFAULT_JOB_FILE_TEMPLATE.sh

```diff
@@ -27,7 +27,8 @@ _galaxy_setup_environment() {
 
 $integrity_injection
 $slots_statement
-export GALAXY_SLOTS
+export _GALAXY_JOB_HOME_DIR
+export _GALAXY_JOB_TMP_DIR
```

Comment:
Shouldn't these 2 go after calling `_galaxy_setup_environment` below?

Reply:
Yes. They should.

Reply:
What do you think should we also document these variables? Maybe here: https://github.com/simonbray/galaxy/blob/dfd0e0b15f70df091099263a4885860b4fe54c36/lib/galaxy/tool_util/xsd/galaxy.xsd#L2830